mandatory/4-tax.js

```diff
@@ -17,7 +21,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(sales) {
+  return  "£" + calculateSalesTax(sales).toFixed(2);
```

Comment:
This is smart how you've condensed it here, but your code should read like a story. It might be a bit clearer to calculate the sales tax into a variable first and then return the priceWithTax so it's clearly named :).